app/vep/vep_resources.py

```diff
@@ -34,10 +36,18 @@ async def submit_vep(request : Request):
     try:
         stream_obj = Streamer(request=request)
         stream_result = await stream_obj.stream()
-        print(stream_obj.parameters.value.decode())
-        print(stream_obj.genome_id.value.decode())
+        parameters = stream_obj.parameters.value.decode()
+        genome_id = stream_obj.genome_id.value.decode()
+
+        parameters_dict = json.loads(parameters)
+        ini_parameters = ConfigIniParams(**parameters_dict)
+        inifile = ini_parameters.create_config_ini_file(stream_obj.temp_dir)
+        vep_config_parameters = VEPConfigParams(vcf=stream_obj.filepath,vep_config=inifile.name)
+        launch_params = LaunchParams(paramsText=vep_config_parameters, labelIds=[])
+        pipeline_params = PipelineParams(launch=launch_params)
         if stream_result:
```

Comment:
Should we add some checks for ini_parameters.create_config_ini_file() 's result here?

Reply:
What sort of check @Mehrnaz-Charkhchi  ?